### ISSUE TITLE
Same probability from fine-tuning custom pre-trained LM

### ISSUE THREAD

**Issue by yuanbit** [huggingface/transformers#3438]
# ❓ Questions & Help

## Details

I have a dataset with QA sentence pairs. I pre-trained a bert model from scractch using run_language_modeling.py where I concatenated the QA pairs and trained the dataset line_by_line.

I tried to fine-tune the custom pre-trained model, but am getting the same probability output for different inputs. I also tried to reduce the learning rate, but the constant probability problem remains.

What could I be doing wrong?

**A link to original question on Stack Overflow**:
